### PR TITLE
Add Clipper Polygon to DEX trades

### PR DESCRIPTION
**Pull request by EdoAPP**
I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

File: polygon/dex/trades/insert_clipper.sql [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    AND dexs.block_time < end_ts
+    ON CONFLICT DO NOTHING
+    RETURNING 1
+)
+SELECT count(*) INTO r from rows;
+RETURN r;
+END
+$function$;
+
+-- fill 2021
```
Comment (aothman) [inline polygon/dex/trades/insert_clipper.sql:102]: Suggest to have two functions here - one to fill 2021 and then another to fill 2022 - present

**Review by aothman — CHANGES_REQUESTED** [review]

**Comment by dsalv** [comment]
Thank you for your contribution @EdoAPP 

Checks:

- [ ] token_a is the token bought
- [ ] basic check vs https://clipper.exchange/data
- [ ] USD amounts also for ETH -> not applicable here

apart from check 1, looks fine to me

**Comment by EdoAPP** [comment]
@dsalv Thanks for catching this. Yes, token_a and token_b were inverted. I've updated the code so token_a is the token bought.

**Review by masquot — APPROVED** [review]
Thank you @EdoAPP, Clipper is now added to DEX trades on Polygon :rocket:

**Pull request merged**